Documentation/ProjectFile/prj/processes/process/SMALL_DEFORMATION/t_deactivated_subdomains.md

```diff
@@ -1,3 +1 @@
-Set of material subdomain indices. The elements in the subdomains by these
-indices will be skipped for local assembly.
-
+Set of material indices, the subdomains by which are excluded from local assembly.
```

Comment:
Maybe better: `Set of material IDs. The subdomains associated to those material IDs are excluded from the local assembly.`

Reply:
Changed accordingly.